doc/api/assert.md

```diff
@@ -82,7 +82,9 @@ changes:
 
 In `strict` mode, `assert` functions use the comparison in the corresponding
 strict functions. For example, [`assert.deepEqual()`][] will behave like
-[`assert.deepStrictEqual()`][].
+[`assert.deepStrictEqual()`][]. This uses the [Strict Equality Comparison][]
```

Comment:
That is actually not correct. The `strict` part just refers to a more precise comparison, not to the used algorithm.
The actual comparison steps are listed at the corresponding entry.

Reply:
Got it! I'm removing this change as it seems confusing. I'll be keeping the other suggestion down below.
Thanks. 👍